src/yml/social.yml

```diff
@@ -0,0 +1,5 @@
+github_username: idvogados
+facebook_username: idvogados
+twitter_username: idvogados
+instagram_username: idvogados
+linkedin_username: idvogados
```

Comment:
Não tenho certeza se já temos uma página ativa no linkedin, mas existe um `idvogados` lá porém como empresa.

O link normal do Jekyll leva para `/in/idvogados` mas a page no linkedin é `/company/idvogados`.
Existe alguma forma de setar esse path manualmente ?

https://www.linkedin.com/company/idvogados